libnavui-maps/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,5 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <string name="mapbox_recenter" tools:ignore="MissingTranslation">Recenter</string>
     <string name="mapbox_route_overview" tools:ignore="MissingTranslation">Overview</string>
+    <string name="mapbox_route_follow" tools:ignore="MissingTranslation">Follow</string>
```

Comment:
@abhishek1508 Not sure if the "Follow" is the right text we should use here. 
Also we need to add all translations for it. Any idea how to do it?

Reply:
@tomaszrybakiewicz Let's use `Following`. Also, I wouldn't worry about translations for now, because we generally get out translations from contributors. I and Minh are working separately to get more contributors to contribute to translations on Transifex.